server/controllers/payroll/multiplePayroll/calculation.js

```diff
@@ -25,4 +25,11 @@ function iprTax(annualCumulation, iprScales) {
   return iprValue;
 }
 
+function automaticRubric(coefficient, variable1, variable2) {
+  const value = variable2 ? coefficient * variable1 * variable2 : coefficient * variable1; 
```

Comment:
This is cool!

One thing that you can do with higher versions of JavsaScript is use rest parameters to make this function smaller (but it might look unclear at first).  It looks like this:
```js
function automaticRubric(coefficient, ...variables) {
  const value = variables.reduce((total, varN) =>total * varN, coefficient);
  return value;
}
```
You can even go smaller by just using:
```js
function automaticRubric(coefficient, ...variables) {
  return variables.reduce((total, next) =>total * next, coefficient);
}
```
What this does is the following:
 1. `...variables` takes all the following function parameters and puts them in an array.  This is called [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters).
 2. `reduce()` will take in a function and a starting number, then return a single number.
 3. `(total, next) => total * next` says that we should take in the total so far, and multiply it by the next number, and return that value.

That means, you can call the function above with:
```js
automaticRubric(1) // => 1
automaticRubric(1, 2) // => 2
automaticRubric(1, 2, 3) // => 6
automaticRubric(1, 23, 7, 8, 12, 30) // => 463680
```
Much more flexible!

(You don't have to do this, just wanted to show you it exists)